NagorniyAleksandr/SQLTask/src/main/java/university/service/ServiceImpl.java

```diff
@@ -0,0 +1,267 @@
+package university.service;
+
+import org.apache.log4j.Logger;
+import university.dao.QueryCreator;
+import university.exceptions.AppDBException;
+import university.exceptions.InvalidQueryParameterException;
+import university.models.*;
+
+import java.util.List;
+
+public class ServiceImpl implements Service {
```

Comment:
Add validation with Regexp
Example:
```addStudent(Student)```

check fields value of the student object
name, age and so on